public/dev.html

```diff
@@ -32,9 +32,9 @@
     />
 
     <!-- Load CSS files -->
-    <link href="css/normalize.css" rel="stylesheet" />
-    <link href="css/skeleton.css" rel="stylesheet" />
-    <link href="css/style.css" rel="stylesheet" />
+    <link href="../client/css/normalize.css" rel="stylesheet" />
+    <link href="../client/css/skeleton.css" rel="stylesheet" />
+    <link href="../client/css/style.css" rel="stylesheet" />
```

Comment:
Do changes like this ensure that the deployed directory (that can be locally tested with `gh-pages` ) works in a **standalone** way?

Reply:
Yeah this is something I am confused about, there does seem to be a relative path in the main example on [Parcel](https://parceljs.org/). 

I guess we need to wait for the CI integration?

Reply:
As a rule of thumb, anything in the folder that is going to be deployed by the GH Pages CI (something which would need to be tweaked in this PR) shouldn't rely on resources outside of it (hence why I initially moved the `models/` to be _inside_ `/public`).

Reply:
I'm not 100% understanding the difference between public and client - if they serve the same purpose (static files that will be served to end users) should we combine them (which will solve this problem as well)?

Reply:
@misterpeddy As discussed in the call (I think when you had your landlord around), one of them (`client`) is where the source files (ESM modules) would be, whilst the other one (`public`) contains the static content (HTML pages and assets).

There's possibly a better structure which would allow having the source files separated from the bundled JS code which should be in the same folder as the static content.